WordPress/src/main/java/org/wordpress/android/widgets/MaterialTextViewWithArabicNumerals.kt

```diff
@@ -0,0 +1,24 @@
+package org.wordpress.android.widgets
+
+import android.content.Context
+import android.util.AttributeSet
+import com.google.android.material.textview.MaterialTextView
+
+/**
+ * MaterialTextView which enforces Western Arabic numerals.
+ */
+class MaterialTextViewWithArabicNumerals @JvmOverloads constructor(
```

Comment:
I like the approach of wrapping the `MaterialTextView` as it provides a cleaner solution 👍 
I'm a bit undecided about the naming `MaterialTextViewWithArabicNumerals` though. I feel that a more generic name (e.g. `MaterialTextViewWithNumerals`) might be more future proof in case we need to replace something else in the future.

Reply:
Good idea, I've renamed the view to `MaterialTextViewWithNumerals` in https://github.com/wordpress-mobile/WordPress-Android/pull/17160/commits/6eb6b4b8bdbef713ff6a7934e95922d8814b14f5. Thank you @antonis 🙇